core/src/main/java/com/novoda/landingstrip/TabsContainer.java

```diff
@@ -57,18 +57,13 @@ boolean isEmpty() {
 
     void startWatching(final ViewPager viewPager, final ViewPager.OnPageChangeListener onPageChangeListener) {
         if (hasTabs()) {
-            getTabAt(0).getViewTreeObserver().addOnGlobalLayoutListener(
-                    new ViewTreeObserver.OnGlobalLayoutListener() {
-                        @Override
-                        public void onGlobalLayout() {
-                            ViewTreeObserver observer = getTabAt(0).getViewTreeObserver();
-                            removeOnGlobalLayoutListener(observer, this);
-                            viewPager.addOnPageChangeListener(onPageChangeListener);
-
-                            onPageChangeListener.onPageScrolled(viewPager.getCurrentItem(), 0, 0);
-                        }
-                    }
-            );
+            viewPager.addOnPageChangeListener(onPageChangeListener);
```

Comment:
So `onGlobalLayout` was getting triggered multiple times (when changing tab etc too), and it was adding observers to tabs that didn't exist. Whereas now it only does it once so it shouldn't get the error we were getting.